app/src/main/java/com/example/masala_food_recipes/data/interactors/Cuisine.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.masala_food_recipes.data.interactors
+
+import com.example.masala_food_recipes.data.entities.Recipe
+
+class Cuisine(private val recipes:List<Recipe>) {
+
+    fun getCuisines():Map<String,Int>{
+        val mapCuisine = mutableMapOf<String,Int>()
+        recipes.forEach {
+            mapCuisine.putIfAbsent(it.cuisine,0)
+            mapCuisine[it.cuisine] = mapCuisine[it.cuisine]!! + 1
+        }
+        return mapCuisine
+    }
+
+    fun getRecipesName():Map<String,List<String>>{
```

Comment:
This function should be in a separated Class Named ForYouRecipes or something else cause it doesn't represent cuisines